assets/scss/hero.scss

```diff
@@ -29,61 +53,85 @@
   }
 
   &.browse-jumbotron {
-    .container {
-      padding-bottom: 3rem;
+    margin-top: 2rem;
```

Comment:
On smaller screens (anything between the mobile view of the header and desktop, like iPad portrait) the header is "clued" to the notification.

Reply:
ok fixed